Xamarin.Forms.Platform.Android/Renderers/EditorRenderer.cs

```diff
@@ -254,7 +254,7 @@ void UpdateText()
 			if (EditText.Text == newText)
 				return;
 
-			EditText.Text = newText;
+			EditText.Text = TrimToMaxLength(newText);
 			EditText.SetSelection(newText.Length);
```

Comment:
You need to updated the SetSelection with the trimmed length not the newText one.

Reply:
sorry... my bad 😅